src/ape/api/providers.py

```diff
@@ -557,9 +561,15 @@ def mine(self, num_blocks: int = 1):
             num_blocks (int): The number of blocks allotted to mine. Defaults to ``1``.
         """
 
-    @cached_property
-    def test_config(self) -> PluginConfig:
-        return self.config_manager.get_config("test")
+    @raises_not_implemented
+    def set_balance(self, address: AddressType, amount: int):
+        """
+        Change the balance of an account.
+
+        Args:
+            address (AddressType): An address on the network.
+            amount (int): The balance to set in the address.
```

Comment:
Can this API support conversions? Or do you think that's better off upstream (e.g. `ChainManager`)?

Reply:
Yeah, I think that is good point because that is how I made the Hardhat one work so you can do stuff like

```python
provider.set_balance(me, "1000 ETH")
```

Reply:
For sure, or it can also be:
```py
chain.set_balance(me, "1000 ETH")
```

Reply:
> does it make more sense in chain then? chain is like the high-level provider sometimes

Yeah, if it's there then you can do the conversion step and leave `ProviderAPI` to only accept integer value